src/features/history/effects.ts

```diff
@@ -50,7 +50,7 @@ export const initializeHistoryBuffer: AppThunk<RootState> = async (
         dispatch(setNumberOfLinesInHistory(historyBuffer.getNumberOfLines()));
         if (dispatch(validateNumberOfLinesInHistory)) {
             logger.warn(
-                'History file is full, consider increasing the Max lines in history file, or cleaning it up, by using the Terminal Settings in the Side Panel.'
+                'History file is full, consider increasing the Max lines in history file, or cleaning it up, by using the Terminal Settings in the Side Panel.' // todo: is that correct message? How to clean it from side panel?
```

Comment:
@aadnekar Please check this todo, I'm not sure message is correct

Reply:
Yes, we removed that feature. So, the message is wrong.

Reply:
![image](https://github.com/NordicSemiconductor/pc-nrfconnect-serial-terminal/assets/34618612/894cf8c4-7721-4a98-8a4a-01cd2d62c972)

Could just warn that the history file is full, and that more info can be found in the Terminal Settings tab?

```suggestion
                'History file is full. You can find more information about the history file in the Terminal Settings tab.'
```

Reply:
Looks good, committed